Enhance intro content (#2733)
astronomer/docs: astro/configure-deployment-resources.md

description: "Manage Deployment settings to customize your Deployment for your use case."
---

> After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its performance. There's two categories of options for configuring your Astro Deployment to best meet your needs: metadata configuration and resource configuration.

Author: ```suggestion
After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its performance. There's two categories of options for configuring your Astro Deployment to best meet your needs: Deployment settings and resource configuration.
```

> Use metadata configurations to manage the administration of your Deployment. This includes includes actions like updating information about your Deployment or transferring it to another Workspace. The following actions allow you to organize and label your Deployments within your Workspace in a way that best fits how your team works with Astro: 

Author: ```suggestion
Use Deployment settings to manage the administration of your Deployment. This includes includes actions like updating information about your Deployment or transferring it to another Workspace. The following actions allow you to organize and label your Deployments within your Workspace in a way that best fits how your team works with Astro: 
```

- Transfer a Deployment to another Workspace in your Organization.
- Delete a Deployment.

You also have configuration options that allow you to customize the resource use, infrastructure, and performance of your Deployment. This means you can optimize how your deployment uses resources, so you can improve the efficeincy of your Deployment processing, optimize compute resources, or enable advanced use cases for workload and cost optimization. The following configuration options enable you to specialize your Deployment configuration to get the maximum performance out of your Astro resources:

- Change the Deployment executor.
- Configure Kubernetes Pod resources.
- Change Scheduler resources.
- Enforce CI/CD Deploys.
- Enable High Availability.
- Update your Airflow configuration.

For advanced Deployment resource configurations, see [Manage Airflow executors on Astro](executors-overview.md) and [Configure worker queues](configure-worker-queues.md).

:::cli

This document focuses on configuring Deployments through the Cloud UI. To configure Deployments as code using the Astro CLI, see [Manage Deployments as code](manage-deployments-as-code.md).

:::

> ## Configure metadata

Author: ```suggestion
## Configure Deployment settings
```

  <meta name="og:description" content="Modify the resource settings of a Deployment to make sure that your tasks have the CPU and memory required to complete successfully." />
</head>

> After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its behavior for your specific needs. 

Author: I think that this page might have outgrown the original intent for it - how do we feel about parsing out the individual sections into separate pages that cover smaller actions or categories of actions?

  <meta name="og:description" content="Modify the resource settings of a Deployment to make sure that your tasks have the CPU and memory required to complete successfully." />
</head>

After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its performance. There's two categories of options for configuring your Astro Deployment to best meet your business and technical needs. 

You can manage the administration of your Deployment, which includes actions like updating information about your Deployment or transferring it to another Workspace. The following actions allow you to organize and label your Deployments within your Workspace in a way that best fits how your team works with Astro and remove resources that you're not using: 

- Update a Deployment name and description.
- Add or delete a Deployment alert email.

> - Transfer a Deployment to another Workspace in your Organization.

Author: Yes I agree, I'm not loving Metadata as the bucket, but I'm trying to keep it out of headings/page titles/rel links so that we can change it without too much disruption or need for redirects.
